analyzers/its/expected/Ember-MM/generic.EmberCore.NMT-{84B2143A-D04F-4262-923D-21AEDF86E2B7}-S2259.json

```diff
@@ -12,6 +155,123 @@
 "endColumn":  34
 }
 }
+},
+{
+"id":  "S2259",
+"message":  "'c.values.FirstOrDefault(Function(y) y.label = saved)' is Nothing on at least one execution path.",
+"location":  {
+"uri":  "sources\Ember-MM\Addons\generic.EmberCore.NMT\dlgNMTMovies.vb",
+"region":  {
+"startLine":  960,
+"startColumn":  42,
+"endLine":  960,
+"endColumn":  94
+}
+}
+},
```

Comment:
TP `value` is a field.
https://github.com/SonarSource/sonar-dotnet/blob/145047ee1ed01316ef730427c7af96f0602ee2d8/analyzers/its/sources/Ember-MM/Addons/generic.EmberCore.NMT/dlgNMTMovies.vb#L945-L960
https://github.com/SonarSource/sonar-dotnet/blob/1711a5f773013925e079d11e32bf0a3f592a6706/analyzers/its/sources/Ember-MM/Addons/generic.EmberCore.NMT/Module.NMT.vb#L220-L226